uspace/app/bdsh/util.c

```diff
@@ -73,3 +73,16 @@ unsigned int cli_set_prompt(cliuser_t *usr)
 
 	return 0;
 }
+
+/*
+ * Returns true if the string is a relative or an absolute path
+ */
+bool is_path(const char *cmd)
```

Comment:
This does not work for relative paths that do not start with './'. So for example, if you copy tetris to /foo/bar/qqq and then cd to /foo, you can run this copy of tetris by typing bar/qqq, but bdsh will still not autocomplete it for you.

Reply:
`you can run this copy of tetris by typing bar/qqq, but bdsh will still not autocomplete it for you`

I cannot reproduce your complain. [Screenshot](https://prnt.sc/mynkes)

When using _without_ ./, ../ or / it neither autocompletes nor executes it from the workingdirectory - which I would deem as the correct behaviour

Reply:
Works for me. Are we on the same revision? Your screenshot says b295720, while mine is 1c481ee.

Reply:
Alright, some silly mistakes on my sides. I was not aware that `make clean` does not reset the revision number. I actually solved this issue already, but I messed up to force push it earlier. The new commit contains it accidentely...